### DISCUSSION TITLE
How should footnotes be handled by `strip-markdown`?

### DISCUSSION THREAD

**Discussion by stefanprobst** [remarkjs/remark#781] [Q&A] [answered]
currently `strip-markdown` does not touch `footnoteReference` and `footnoteDefinition` nodes.

i think it might be a good idea to remove them, like e.g. tables are removed?

[these would be the necessary changes](https://github.com/stefanprobst/strip-markdown/commit/f70484f9189865b453ed90df58b3b2276164bf11) (incl. enabling tests for gfm), but i thought i'd ask first, in case the current behavior is intentional.

**Comment by wooorm** [comment, marked as answer]
Yeah, stripping them probably makes the most sense I think

**Reply by stefanprobst** [reply]
ok, cool i'll open a pr then!
